proto/api.proto

```diff
@@ -143,6 +143,13 @@ message SimOptions {
 	bool use_labeled_rands = 9; // Use test level RNG.
 }
 
+enum SpellType {
+	SpellTypeCast = 0;
```

Comment:
The recommended convention for proto enums is to always reserve 0 for "Unknown" (in this case `SpellTypeUnknown`) so that there's always a clear detection for the field not being set in a message (default 0). We've adopted this convention for the bulk of our protos; the only major exception is `Stat` and `PseudoStat`, since those are used to index arrays and we don't want to carry extra array indices around.